### PR TITLE
script standartization cities - aldebaran part 1

### PR DESCRIPTION
**Pull request by Ridley8819**
npc\cities\aldebaran.txt
(from now on, as Haru suggested, code edits come in a separate commit)

---

This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1481)

### CODE REVIEW

File: npc/cities/aldebaran.txt [hunk 1]
```diff
@@ -1173,220 +1171,220 @@ aldeba_in,88,161,3	script	Kafra#05	4_F_KAFRA3,{
...
-		mes "[Kafra]";
-		mes "You've won the 4th prize.";
-		if(@Lotto == 1) getitem Sweet_Potato,100;//Items: Sweet_Potato,
-		if(@Lotto == 2) getitem Red_Potion,150;//Items: Red_Potion,
-		if(@Lotto == 3) getitem Orange_Potion,150;//Items: Orange_Potion,
-		if(@Lotto == 4) getitem Blue_Potion,5;//Items: Blue_Potion,
-		if(@Lotto == 5) getitem Blue_Potion,10;//Items: Blue_Potion,
+		mes("[Kafra]");
+		mes("You've won the 4th prize.");
+		if (@Lotto == 1) getitem(Sweet_Potato, 100);//Items: Sweet_Potato,
```
Comment (dastgir) [inline npc/cities/aldebaran.txt]: 1) getitem in new line.
2) Comment of item name no longer needed, as constants are used

File: npc/cities/aldebaran.txt [hunk 2]
```diff
@@ -1173,220 +1171,220 @@ aldeba_in,88,161,3	script	Kafra#05	4_F_KAFRA3,{
...
+	next();
+	mes("[Lottery Machine]");
+	mes("(rumble~rumble~rumble~)...");
+	next();
 	@temp = rand(10);
 	--@input;
-	if(@input <= 0) return;
-	goto sF_Spin;
+	if (@input <= 0) return;
+	goto(sF_Spin);
```
Comment (dastgir) [inline npc/cities/aldebaran.txt]: use callsub?

File: npc/cities/aldebaran.txt [hunk 3]
```diff
@@ -1173,220 +1171,220 @@ aldeba_in,88,161,3	script	Kafra#05	4_F_KAFRA3,{
...
+	mes("Number of spins remaining: "+@input);
+	next();
+	mes("[Lottery Machine]");
+	mes("(rumble~rumble~rumble~)...");
+	next();
 	@temp = rand(10);
 	--@input;
-	if(@input <= 0) return;
-	goto sF_Spin;
+	if (@input <= 0) return;
```
Comment (dastgir) [inline npc/cities/aldebaran.txt]: return on new line

File: npc/cities/aldebaran.txt [hunk 4]
```diff
@@ -1173,220 +1171,220 @@ aldeba_in,88,161,3	script	Kafra#05	4_F_KAFRA3,{
...
-		mes "You've won the 4th prize.";
-		if(@Lotto == 1) getitem Sweet_Potato,100;//Items: Sweet_Potato,
-		if(@Lotto == 2) getitem Red_Potion,150;//Items: Red_Potion,
-		if(@Lotto == 3) getitem Orange_Potion,150;//Items: Orange_Potion,
-		if(@Lotto == 4) getitem Blue_Potion,5;//Items: Blue_Potion,
-		if(@Lotto == 5) getitem Blue_Potion,10;//Items: Blue_Potion,
+		mes("[Kafra]");
+		mes("You've won the 4th prize.");
+		if (@Lotto == 1) getitem(Sweet_Potato, 100);//Items: Sweet_Potato,
+		if (@Lotto == 2) getitem(Red_Potion, 150);//Items: Red_Potion,
```
Comment (dastgir) [inline npc/cities/aldebaran.txt]: Use else if?

**Comment by Ridley8819** [comment]
@Dastgir
Sorry for late reply, didn't see it. 
I talked about it with haru before doing the commit. He wants 1 commit for pure conversion, and actually script edits in a separate commit (I wrote that in the merge message)

**Comment by Smokexyz** [comment]
@HerculesWS/developers PR has conflicting files. Inactive for over a month, would @Ridley8819 like to update or shall this be closed?

**Pull request closed without merging**